src/open_inwoner/accounts/tests/test_oidc_views.py

```diff
@@ -62,18 +69,134 @@ def test_existing_email_updates_user(
             callback_url, {"code": "mock", "state": "mock"}
         )
 
+        self.assertRedirects(
+            callback_response, reverse("admin:index"), fetch_redirect_response=True
+        )
+
         user.refresh_from_db()
 
+        self.assertTrue(User.objects.filter(oidc_id="some_username").exists())
+        self.assertEqual(user.oidc_id, "some_username")
+
+        db_user = User.objects.filter(oidc_id="some_username").first()
```

Comment:
In general tests are be harder if instead of .filter()/.first() (or .exists()) for single records to just use a .get() so it will also communicate and test the record must exists and cannot be multiple.

Also the test itself seems weird? I don't quite get what it tries to prove: isn't `user` the same record as `db_user`?

Reply:
You're right, the tests were a bit messy already and I was adding stuff around that :sweat_smile:.

I cleaned it up, the test is supposed to verify that an existing admin user (that is not yet OIDC), is updated if a user with the same email logs in with OIDC